### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by duledu**
Footer wip, alsmost done

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,88 +6,149 @@
...
-</footer>
-
-<script src="bootstrap/js/jquery-3.4.1.js"></script>
-<script src="bootstrap/js/bootstrap.min.js"></script>
-<script src="scripts/main.js"></script>
+	<footer class="container-fluid pb-7 pb-lg-3">
+	  	<div class="container">
+	  		<div class="row px-2 px-lg-0 mb-3">
+	  			<div class="col-lg-3 py-lg-3 px-0 px-lg-2 separator-none-lg separator-b">
+	  				<h5 class="text-uppercase"><a id="test"  href="#" class="nav-link d-flex collapsed"
```
Comment (psdi) [inline index.html:19]: @duledu The navs shouldn't be collapsed unless it's in mobile, I think.

File: index.html [hunk 2]
```diff
@@ -6,88 +6,149 @@
...
+	  					<li class="nav-item"><a class="nav-link" href="#" title="University Policies">University Policies</a></li>
+
+	  				</ul>
+						<hr>
+	  			</div>
+	  		</div>
+				</div>
+				</div>
+	  		<div class=" foll-us row px-2 pb-3">
+	  			<h5>FOllOW US</h5>
```
Comment (psdi) [inline index.html]: @duledu The two letter 'l's are lowercase 😜

File: index.html (hunk 1 above)
Comment (psdi) [reply index.html:19]: Also, please change the thickness of the nav links from bold to normal.

Comment (duledu) [reply index.html:19]: I checked it for collapse on the original. It should be collapsed and if you look at their code you will see that the collapse is set to the higher resolution too.

File: index.html (hunk 2 above)
Comment (duledu) [reply index.html]: corrected the typing error

File: index.html (hunk 1 above)
Comment (duledu) [reply index.html:19]: I change the thickness of the nav links.

**Review by psdi — APPROVED** [review]

**Pull request merged**